packages/metrics/src/features/registration/pointGenerator.ts

```diff
@@ -234,7 +234,7 @@ const generatePointLocations = async (
   // tslint:disable-next-line no-increment-decrement
   for (let index = 4; index > 1; index--) {
     locationID = await fetchParentLocationByLocationID(locationID, authHeader)
-    if (!locationID) {
+    if (!locationID || locationID === 'Location/0') {
```

Comment:
Noticed we got a lot of 404 errors from Hearth, that was because of us requesting data for this location. Saves us from doing those requests if nothing else :)